docs/components/defining-components.md

```diff
@@ -39,17 +39,12 @@ type Component interface {
 }
 ```
 
-The completed `impl/your_filename.go` looks like this:
-```go
-package config
+The Component interface is implemented in the file ``comp/<bundleName>/<component>/<component>impl/<component>.go`. Also, the `Module` definition. 
```

Comment:
```suggestion
The Component interface is implemented in the file `comp/<bundleName>/<component>/<component>impl/<component>.go` and the `Module` definition. 
```
The second sentence in this line is a fragment. I made my best guess as to the meaning and fixed it. Please make further edits if I missed the intended meaning.